internal/users/manager.go

```diff
@@ -415,13 +418,24 @@ func getActiveUsers(procDir string) (activeUsers map[string]struct{}, err error)
 	return activeUsers, nil
 }
 
-// GenerateID generates an integer number based on the provided string lowercased.
+// GenerateID deterministically generates an ID between from the given string, ignoring case. The ID is in the range
+// 60000 (the default value of UID_MAX, i.e. the maximum UID for regular users) and MaxInt32.
 func GenerateID(str string) int {
-	lowerCased := strings.ToLower(str)
-	var sum int
-	for i, c := range lowerCased {
-		// Multiplies the value of the rune (which is >0) by its index+1. Subtracts the index to add another layer of conflict prevention.
-		sum += int(c)*(i+1) - i
-	}
-	return (sum % (100000 - 65537)) + 65536 // Ensures that ID is between 65536 and 100000
+	const minID = 60000
+	const maxID = math.MaxInt32 // use MaxInt32 for 32-bit systems
```

Comment:
can you replace this note that we want MaxInt32 to workaround possible issues in some softwares when the uid/gid is higher than this? (this is not for 32-bit systems)

Reply:
Thanks!